scripts/commands/gotonpc.lua

```diff
@@ -0,0 +1,25 @@
+---------------------------------------------------------------------------------------------------
+-- func: gotonpc
+-- auth: Chronos77
+-- desc: Goes to the target player.
+---------------------------------------------------------------------------------------------------
+
+cmdprops =
+{
+    permission = 1,
+    parameters = "s"
+};
+
+function onTrigger(player, target)
+    if (target == nil) then
+        player:PrintToPlayer("You must enter a valid npc id.");

```

Comment:
Thanks for your return. I corrected this.

Yes I am currently using because I find it easier to write an id as writing a position no ?
Basically I wanted to do with the name of npc but we have many npc with the same name.